lib/queue.js

```diff
@@ -526,7 +527,7 @@ class Queue extends Emitter {
         ? job.options.backoff.strategy
         : 'immediate';
       const strategy = job.options.retries > 0
-        ? backoff.get(strategyName)
+        ? this.backoffStrategies.get(strategyName)
```

Comment:
So the behavior here if the backoff strategy hasn't been configured (maybe due to deployment issues in multi-service scenarios) is that the job would get retried immediately? That may be a suboptimal result - not sure. It also seems like we should report this as an error for operational visibility. Not sure if that should be an `error` event or something else. Thoughts?

Reply:
tbh can't remember what I was thinking. seems like this matches current behavior, albeit with a little more "buyer beware."